ilc/client/Client.js

```diff
@@ -41,6 +40,9 @@ import ErrorHandlerManager from './ErrorHandlerManager/ErrorHandlerManager';
 
 import { FRAGMENT_KIND } from '../common/constants';
 import { SdkFactoryBuilder } from "./Sdk/SdkFactoryBuilder";
+import {TransitionHooks} from './TransitionManager/TransitionHooks/TransitionHooks';
```

Comment:
seems like formatting is not unified across the file

Reply:
populated to registry and ilc folders